index.html

```diff
@@ -2,21 +2,57 @@
 layout: default
 title: Home
 ---
-{% for post in site.categories.announcements limit: 1 %}
 
-<section class="section--center mdl-grid mdl-grid--no-spacing mdl-shadow--2dp">
-  <div class="mdl-card mdl-cell mdl-cell--12-col">
-    <div class="mdl-card__supporting-text">
-      <h4>{{ post.title }}</h4>
-      <h6>{{ post.date | date_to_long_string }}</h6>
-        {{ post.content }}
-        <hr />
-        <em>{{ post.author }}</em>
-      </div>
+<section id="home-about" class="jumbotron jumbotron-fluid">
+    <div class="container">
+        <h1 class="display-4">Hello, (RIT|World)!</h1>
+        <p class="lead">RITlug is a student-led organization which aims to promote and teach students about Linux &amp; open source.</p>
+        <hr class="lead-4" />
+        <p>RITlug has weekly meetings and events where we work on projects and discuss interesting topics in the Linux &amp; open source communities.</p>
+        <p>Meetings every {{ site.ritlug-day }}, {{ site.ritlug-time }} in {{ site.ritlug-place }}, Fall &amp; Spring semesters.</p>
     </div>
-  <div class="mdl-card__actions mdl-card--border">
-    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href="/announcements">Previous Announcements</a>
-  </div>
 </section>
-{% endfor %}
-<br />
+
+<section id="do-stuff" class="container">
+    <div class="row" style="margin-bottom:-1.2em;">
+        <div class="col-12 col-md-6" style="margin-bottom:1.2em;">
+            <h2>Discuss</h2>
+            <div class="list-group">
+                <a href="https://rit-lug.slack.com/" class="list-group-item list-group-item-action">Slack (needs <code>@rit.edu</code> email)</a>
+                <a href="https://telegram.me/ritlugclub" class="list-group-item list-group-item-action">Telegram</a>
```

Comment:
When we started the Slack, we made a decision to deprecate the Telegram group and transition away from promoting it in the club. I think it is better to be opinionated on where we point people to join and participate with the club, because:

1. Too many options can be too confusing for where to join or participate
2. Slack integrates multiple parts of our community and club together in one place (FOSS, projects, announcements channel, etc.).

Since we retired the Telegram group as an unofficial platform, I think we shouldn't mention it on the home page.

Reply:
Sure, I can do that. Do you have any suggestions for what I could put there instead so I can keep it visually balanced?

Reply:
I'm slightly hesitant to do that due to the number of email-related issues we've had lately. I think this should be an eboard discussion

Reply:
(continuing this in main comments thread)

Reply:
@ct-martin Email is still a critical part of how we communicate out with the announcements we post to the website. What alternative are you proposing?

Reply:
@jwflory We could use CampusGroups or another system like MailChimp or maybe a NameCheap list. If we have something that works reliably I don't mind putting it there, however, we've had half a dozen to a dozen issues with mail this past semester alone that are of a level to concern me. If it's "just a way" of communicating I don't mind it too much, but if we're really promoting it, it should work.